pkg/common/peertracker/tracker_linux.go

```diff
@@ -106,10 +132,14 @@ func (l *linuxWatcher) IsAlive() error {
 	// TODO: Evaluate the use of `starttime` as the primary exit detection mechanism.
 	currentStarttime, err := getStarttime(l.pid)
 	if err != nil {
-		return fmt.Errorf("caller exit suspected due to failure to get starttime: %w", err)
+		msg := "caller exit suspected due to failure to get starttime"
+		l.log.WithError(err).Warn(msg)
+		return errors.New(msg)
```

Comment:
Can we retain the error from the call to `getStarttime()` that was previously wrapped in the new error we pass back to the caller?

Reply:
sure. done